js/samples/02.teams-features/b.adaptiveCards.typeAheadBot/src/index.ts

```diff
@@ -128,13 +128,16 @@ app.adaptiveCards.search('npmpackages', async (context: TurnContext, state: Turn
     return npmPackages;
 });
 
+interface DynamicSubmitData {
+    dynamicSelect?: string;
+}
 interface SubmitData {
```

Comment:
maybe remove if we don't need this interface anymore

Reply:
Both are being used